docs/reference/configurations.md

```diff
@@ -0,0 +1,87 @@
+# Configurations
+
+- _blog_hostname_: Hostname for accessing WordPress, if ingress relation is active. Defaults to the
```

Comment:
What is the reason for not putting this on the configuration options? On charmhub, there is a default page for the configurations: https://charmhub.io/wordpress-k8s/configure?channel=edge. The same comment applies for actions

Reply:
What do you think about removing the `configurations.md` and `actions.md` files altogether and just updating the `actions.yaml` & `config.yaml`? The way users will interact with the documentation is through the sidebar and if the desired action is to guide them towards correct tabs, I think it may be better to not have these two sections at all.
With reference to Indico charm, when the users to land on [reference/actions](https://charmhub.io/indico/docs/reference-actions), the only content is See Actions with a navlink to the actions tab which I think is not the best UX.

Reply:
Resolved to adding a reference link to the according tabs as per discussion today. Thanks for the comment!

Reply:
As discussed, we will use the Indico method of having a document with just a link so that there is a navigation table entry until `upload-charm-docs` supports external links preperly